projects/ids-enterprise-ng/src/lib/swaplist/soho-swaplist.component.ts

```diff
@@ -101,21 +132,21 @@ export class SohoSwapListComponent implements AfterViewInit, OnDestroy {
   private _showFullAccessCard = false;
```

Comment:
From a stylistic side are we consistently naming the variables that are internal with an underscore? I feel like its mixed and probably should just drop the underscore @bthharper  @pwpatton ?

Reply:
Yes - no underscores (or Hungarian notation).